react-backend/client/src/components/visitors/signup_form.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import Input from './input';
+import Select from './select';
+
+export default props => (
+  <div>
+    <form className="SignupForm" onChange={props.handleChange}>
+      <section className="Main col-9">
+        <h1>Please tell us about yourself</h1>
+        {props.error && (
+          <div className="ErrorText">
+            {props.error.map(el => <span key={el}>{el}</span>)}
+          </div>
+        )}
+        <Input question="Name" option="fullname" />
+        <Input question="Email" option="email" />
```

Comment:
Any reason we're not using the `required` attribute on non-optional form inputs?